frontend/src/components/receipts/ReceiptsIncludedInBlock.tsx

```diff
@@ -0,0 +1,17 @@
+import { FC } from "react";
+
+import { useWampSimpleQuery } from "../../hooks/wamp";
+import ReceiptsList from "./ReceiptsList";
+
+interface Props {
+  blockHash: string;
+}
+
+const ReceiptsIncludedInBlock: FC<Props> = ({ blockHash }) => {
+  const receiptsList = useWampSimpleQuery("receipts-list-by-block-hash", [
```

Comment:
Let's rename the wamp endpoint to `included-receipts-list-by-block-hash` to avoid confusion in the future